apps/delivery/migrations/0001_initial.py

```
@@ -0,0 +1,34 @@
+# Generated by ChatGPT on 2024-XX-XX
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        ('appointments', '0002_initial'),
+    ]
```

Ruff (0.14.0)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)